src/groupchatdlg.cpp

```diff
@@ -2481,11 +2481,11 @@ void GCMainDlg::buildMenu()
     // Dialog menu
     d->pm_settings->clear();
 
-    d->actions->action("gchat_info")->addTo(d->pm_settings);
-    d->actions->action("gchat_clear")->addTo(d->pm_settings);
-    d->actions->action("gchat_configure")->addTo(d->pm_settings);
+    d->pm_settings->addAction(d->actions->action("gchat_info"));
+    d->pm_settings->addAction(d->actions->action("gchat_clear"));
+    d->pm_settings->addAction(d->actions->action("gchat_configure"));
     //#ifdef WHITEBOARDING
-    //    d->act_whiteboard->addTo( d->pm_settings );
+    //    d->pm_settings->addAction(d->actions->action("act_whiteboard"));
```

Comment:
If you review the remaining code in this file you will see the comment is not wrong.